core/server/models/post.js

```diff
@@ -587,12 +587,12 @@ Post = ghostBookshelf.Model.extend({
 
         // CASE: if the filter contains an `IN` operator, we should return the posts first, which match both tags
         if (options.filter && options.filter.match(/(tags|tag):\s?\[.*\]/)) {
-            order = `count(tags.id) DESC, ${order}`;
+            order = `(SELECT count(*) FROM posts_tags WHERE post_id = posts.id) DESC, ${order}`;
         }
 
         // CASE: if the filter contains an `IN` operator, we should return the posts first, which match both authors
         if (options.filter && options.filter.match(/(authors|author):\s?\[.*\]/)) {
-            order = `count(authors.id) DESC, ${order}`;
+            order = `(SELECT count(*) FROM posts_authors WHERE post_id = posts.id) DESC, ${order}`;
```

Comment:
We need a TODO here if this SELECT works.

mongo-knex no longer joins with e.g. authors table, so we dont have access to authors.id.
AND even if mongo-knex would still join, then it would tie mongo-knex and Ghost together, because Ghost knows that mongo-knex will JOIN 👻 So raw SQL is fine here? needs investigation.

This is a little lower priority for now, because it only kicks in if you use tags:[X,Y].
But we should look at it soon.

Reply:
@gargol from our sync :)

Reply:
@gargol I am leaving soon. Would be cool if you could add a code comment :) Thanks